### PR TITLE
Release Notes

### PR DESCRIPTION
**Pull request by Firstyear**
# Change summary

- Update release notes for 1.4.0

Checklist

- [x] This PR contains no AI generated code
- [ ] book chapter included (if relevant)
- [ ] design document included (if relevant)

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -14,6 +14,57 @@ report it to our [issue tracker].
...
+your own domain and OAuth2 client icons.
+- OAuth2 strict redirect uri is now required. Ensure you have read
+[our upgrade documentation](https://github.com/kanidm/kanidm/blob/master/book/src/server_updates.md#general-update-notes).
+and taken the needed steps before upgrading.
+
+### 1.4.0 Release Highlights
+
+- Improve handling of client timeouts when the server is under high load
+- Resolve a minor issue preventing some credential updates from saving
+- pam/nss unixd now allow non-kanidm backends - more to come soon
```
Comment (yaleman) [inline RELEASE_NOTES.md:43]:
```suggestion
- PAM/NSS unixd now allow non-Kanidm backends - more to come soon
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -14,6 +14,57 @@ report it to our [issue tracker].
...
+- Admin CLI tools support instance profiles allowing admin of multiple sites to be easier
+- Resolve a minor issue in OAuth2 introspection which returned the wrong claim for `token_type`
+- Resolve an issue where memberOf should imply dynamicMemberOf in access controls
+- Allow configuration of custom domain icons
+- Internal representation of attributes changed to an enum to reduce memory consumption
+- Add CreatedAt and ModifiedAt timestamps to entries
+- Expose RFC7009 and RFC7662 via OIDC metadata discovery
+- Improve pipe handling for CLI tools
+- Large techdebt cleanups
+- pam/nss unixd can provide system users, replacing `pam_unix`
```
Comment (yaleman) [inline RELEASE_NOTES.md:55]:
```suggestion
- PAM/NSS unixd can provide system users, replacing `pam_unix`
```

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -14,6 +14,57 @@ report it to our [issue tracker].
...
+- Account policy supports LDAP password fallback to main password
+- pam/nss unixd can extend a system group with members from remote sources (such as Kanidm)
```
Comment (yaleman) [inline RELEASE_NOTES.md:57]:
```suggestion
- PAM/NSS unixd can extend a system group with members from remote sources (such as Kanidm)
```

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -14,6 +14,57 @@ report it to our [issue tracker].
...
+- Resolve a potential issue in replication on upgrade where migrated entries cause a referential
+  integrity conflfict leading to a forced intialisation
```
Comment (yaleman) [inline RELEASE_NOTES.md:59]:
```suggestion
  integrity conflict leading to a forced initialisation
```

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -14,6 +14,57 @@ report it to our [issue tracker].
...
+- Display credential reset token expiry time when created on cli
```
Comment (yaleman) [inline RELEASE_NOTES.md:60]:
```suggestion
- Display credential reset token expiry time when created on CLI
```

**Review by yaleman — CHANGES_REQUESTED** [review]
Just me being me

**Review by yaleman — APPROVED** [review]

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -14,6 +14,57 @@ report it to our [issue tracker].
...
+- PAM/NSS unixd can extend a system group with members from remote sources (such as Kanidm)
+- Resolve a potential issue in replication on upgrade where migrated entries cause a referential
+  integrity conflict leading to a forced initialisation
+- Display credential reset token expiry time when created on CLI
+- Reload certificates and private keys on SIGHUP
+- Remove a large number of dependencies that were either not needed or could be streamlined
+- SCIM foundations for getting and modifying entries, reference handling, and complex attribute
+  display. Much more to come in this space!
+- Rewrite the entire web frontend to be simpler and faster, allowing more features to be added
+  in future. Greatly improves user expirence as the pages are now very fast to load!
```
Comment (gbw) [inline RELEASE_NOTES.md:66]: Small typo s/expirence/experience/

Comment (gbw) [reply RELEASE_NOTES.md:66]: in _the_ future

Comment (Firstyear) [reply RELEASE_NOTES.md:66]: Will fix in a diff PR.

**Pull request merged**